esmvaltool/recipes/recipe_radiation_budget.yml

```
@@ -0,0 +1,129 @@
+# ESMValTool
+# recipe_radiation_budget.yml
+---
+documentation:
+  title: Radiation Budget
+  description:
+    This diagnostic analyses the radiation budget by separating top-of-atmosphere
+    fluxes into clear-sky and cloud forcing components, and surface fluxes into
+    downwelling and upwelling components. Model predictions are compared against
+    three observational estimates, one of which (Stephens et al, 2012) includes
+    uncertainty estimates. When the black error bars overlap the zero line, the
+    model is consistent with observations according to Stephens et al.
```

For consistency with what we did in the documentation.
```suggestion
    three observational estimates, one of which (Stephens et al. 2012) includes
    uncertainty estimates. When the black error bars overlap the zero line, the
    model is consistent with observations according to Stephens et al. 2012.
```